frontend/src/metabase/query_builder/components/AlertListPopoverContent.jsx

```diff
@@ -0,0 +1,254 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { getQuestionAlerts } from "metabase/query_builder/selectors";
+import { getUser } from "metabase/selectors/user";
+import { deleteAlert, unsubscribeFromAlert } from "metabase/alert/alert";
+import { AM_PM_OPTIONS, DAY_OF_WEEK_OPTIONS, HOUR_OPTIONS } from "metabase/components/SchedulePicker"
+import Icon from "metabase/components/Icon";
+import Modal from "metabase/components/Modal";
+import { CreateAlertModalContent, UpdateAlertModalContent } from "metabase/query_builder/components/AlertModals";
+import _ from "underscore";
+import cx from "classnames";
+import cxs from 'cxs';
+
+const customSpacing = cxs ({
+    marginLeft: '10px'
+})
+const ownAlertCustomSpacing = cxs ({
+    marginLeft: '9px',
+    marginRight: '17px'
+})
+const unsubscribeButtonSpacing = cxs ({
+    transform: `translateY(4px)`
+})
+
+@connect((state) => ({ questionAlerts: getQuestionAlerts(state), user: getUser(state) }), null)
+export class AlertListPopoverContent extends Component {
+    state = {
+        adding: false,
+        ownAlertRemovedAsNonAdmin: false
+    }
+
+    onAdd = () => {
+        this.props.setMenuFreeze(true)
+        this.setState({ adding: true })
+    }
+
+    onEndAdding = () => {
+        this.setState({ adding: false })
+    }
+
+    onRemovedOwnAlert = () => {
+        this.setState( { ownAlertRemovedAsNonAdmin: true })
+    }
+
+    isCreatedByCurrentUser = (alert) => {
+        const { user } = this.props;
+        return alert.creator.id === user.id
+    }
+
+    render() {
+        const { questionAlerts, setMenuFreeze, user } = this.props;
+        const { adding, ownAlertRemovedAsNonAdmin } = this.state
+
+        const isNonAdmin = !user.is_superuser
+        const [ownAlerts, othersAlerts] = _.partition(questionAlerts, this.isCreatedByCurrentUser)
+        // user's own alert should be shown first if it exists
+        const sortedQuestionAlerts = [...ownAlerts, ...othersAlerts]
+        const hasOwnAndOthers = ownAlerts.length > 0 && othersAlerts.length > 0
+
+        return (
+            <div style={{ minWidth: 410 }}>
```

Comment:
Any reason we're using `style` instead of `cxs` here?

Reply:
I think this was already in there before I gained true cxs enlightenment. I'll change this.